sdk/appconfiguration/app-configuration/README.md

```diff
@@ -1,28 +1,91 @@
-# Azure App Configuration client library for JS
+# App Configuration client library for TypeScript/JavaScript
 
-Azure App Configuration is a managed service that helps developers
-centralize their application configurations, simply and securely.
+[Azure App Configuration][appconfig_concepts] is a managed service that helps developers centralize their application and feature settings simply and securely.
 
-This client library lets you create, retrieve, update, and delete 
-settings within Azure App Configuration.
+Use the client library for App Configuration to:
+* Create flexible key representations and mappings
+* Tag keys with labels
+* Replay settings from any point in time
 
-* [Azure App Configuration documentation](https://docs.microsoft.com/en-us/azure/azure-app-configuration/)
-* [NPM](https://www.npmjs.com/package/@azure/app-configuration)
-* [Source](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/appconfiguration/app-configuration/)
-* [Samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/appconfiguration/app-configuration/samples)
+[Source](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/appconfiguration/app-configuration/) | [NPM](https://www.npmjs.com/package/@azure/app-configuration) | [API Reference documentation](https://azure.github.io/azure-sdk-for-js/app-configuration/) | [Samples][samples]
 
 ## Getting started
 
-### Currently supported environments
-
-- Node.js version 8.x.x or higher
-
-### How to Install
+### 1. Install the `@azure/app-configuration` package
```

Comment:
The other readmes list prereqs for resources required right before this header in this format:

```
**Prerequisites**: You must have an [Azure Subscription](https://azure.microsoft.com) and an [App Configuration resource](https://docs.microsoft.com/en-us/azure/azure-app-configuration/) to use this package. This package currently
only supports NodeJS versions higher than 8.0.0.
```